modular_zubbers/code/modules/cargo/expressconsole.dm

```diff
@@ -22,7 +22,8 @@
 	COMPANY_NAME_DEFOREST_MEDICAL,
 	COMPANY_NAME_NRI_SURPLUS,
 	COMPANY_NAME_BLACKSTEEL_FOUNDATION,
-	COMPANY_NAME_NAKAMURA_ENGINEERING_MODSUITS
+	COMPANY_NAME_NAKAMURA_ENGINEERING_MODSUITS,
+	COMPANY_NAME_SAR_INC
```

Comment:
The extra comma isn't a meme it makes future conflict resolution and gitblaming faster
```suggestion
	COMPANY_NAME_SAR_INC,
```